src/styles/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding-block: 32px 16px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  align-items: center;
+
+  &__product {
+    display: flex;
+    &-image {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__details {
+    margin: 40px 0 0;
+    &-name {
+      width: 166px;
+      font-size: 12px;
+      color: $primary-color-text-card;
```

Comment:
The variable `$primary-color-text-card` is used here but not defined. Ensure this variable is defined in your `variables.scss` file to avoid compilation errors.